migrations/4_initialize_contracts.js

```diff
@@ -66,5 +66,5 @@ module.exports = async function (callback) {
   }
 
   console.log('Contracts initialized');
-  callback(null);
+  // callback(null);
```

Comment:
Can we leave a comment here to say that this null is needed and why? I can remember is part of truffle, but to be sure.

Reply:
Totally agree on that, I also have it commented, otherwise Truffle throws an error.